src/Microsoft.AspNet.Mvc.Core/OutputFormatter.cs

```diff
@@ -23,6 +23,19 @@ public abstract class OutputFormatter : IOutputFormatter, IApiResponseFormatMeta
         private readonly List<MediaTypeHeaderValue> _supportedMediaTypes;
 
         /// <summary>
+        /// Returns UTF8 Encoding without BOM and throws on invalid bytes.
+        /// </summary>
+        public static readonly Encoding UTF8EncodingWithoutBOM
```

Comment:
My question was is there any utility in setting this for an output formatter? Seems like it's useful specifically for decoding, not encoding.